rotkehlchen/api/rest.py

```diff
@@ -2476,10 +2476,13 @@ def delete_eth2_validator(
         return api_response(result, status_code=status_code)
 
     @async_api_call()
-    def get_ethereum_airdrops(self) -> dict[str, Any]:
+    def get_airdrops(self) -> dict[str, Any]:
+        with self.rotkehlchen.data.db.conn.read_ctx() as cursor:
+            evm_accounts = self.rotkehlchen.data.db.get_evm_accounts(cursor)
```

Comment:
you don't need to store the variable i guess. the try/except block can just be indented